src/test/java/webserver/data/JsonClientTest.java

```diff
@@ -1,12 +1,14 @@
-package webserver.data;
+                              package webserver.data;
```

Comment:
Fell asleep on the keyboard? _he says, from experience_

Reply:
haha. oops. no idea!